tests/image/Dockerfile

```diff
@@ -0,0 +1,108 @@
+# docker build -t registry.gitlab.com/altinity-qa/clickhouse/cicd/clickhouse-operator .
+FROM ubuntu:20.04
+
+RUN apt-get update \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install --yes \
+    apt-transport-https \
+    software-properties-common \
+    vim \
+    gettext-base \
+    ethtool \
+    jq \
+    socat \
+    gpg \
+    ca-certificates \
+    bash \
+    btrfs-progs \
+    e2fsprogs \
+    iptables \
+    xfsprogs \
+    tar \
+    pigz \
+    wget \
+    git \
+    perl \
+    iproute2 \
+    cgroupfs-mount \
+    python3-pip \
+    tzdata \
+    psmisc \
+    libreadline-dev \
+    libicu-dev \
+    bsdutils \
+    curl \
+    liblua5.1-dev \
+    luajit \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    gdb \
+    && rm -rf \
+        /var/lib/apt/lists/* \
+        /var/cache/debconf \
+        /tmp/* \
+    && apt-get clean
+
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
+    && echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
+    && apt-get update \
+    && apt-get install --yes docker-ce docker-ce-cli containerd.io
+
+RUN curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg \
+    && echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list \
+    && curl -LO https://storage.googleapis.com/minikube/releases/v1.23.0/minikube-linux-amd64 \
+    && install minikube-linux-amd64 /usr/local/bin/minikube \
+    && apt-get update \
+    && apt-get install -y kubectl
+
+ENV TZ=Europe/Moscow
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+COPY requirements.txt /home/master/requirements.txt
+RUN pip3 install -U -r /home/master/requirements.txt
+
+ENV GOLANG_VERSION 1.16
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F6BC817356A3D45E \
+    && add-apt-repository ppa:longsleep/golang-backports \
+    && apt install --no-install-recommends -y golang-${GOLANG_VERSION}-go \
+    && ln -nvsf /usr/lib/go-${GOLANG_VERSION}/bin/go /bin/go \
+    && ln -nvsf /usr/lib/go-${GOLANG_VERSION}/bin/gofmt /bin/gofmt
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CC86BB64 \
```

Comment:
and again, why we need yq in container? do we manipulate yaml files during tests?

Reply:
I don't remember exactly when, but there was an issue that required installing `yq`